basilisk/src/ast/interpreter/dimension-tests/test13.c

```
@@ -0,0 +1,16 @@
+/**
+Check that 'datasize' and other field properties are defined properly
+even when allocating temporary fields within undefined
+conditionals. */
+
+int main()
+{
+  init_grid (1);
+  scalar s[];
+  int cond;
+  if (cond) {
```

Cppcheck (2.10-2)

[error] Uninitialized variable

(uninitvar)